courses/ocp4_advanced_deployment/lab_04_1/solve_lab.yml

```diff
@@ -52,7 +52,7 @@
 
     - name: Wait for nodes to come online
       pause:
-        seconds: 300
+        seconds: 500
```

Comment:
I'm not a fan of time passing without checks running.  Perhaps increase the retries in the next play?  Up to you.  It worked for me with 300 seconds.

Reply:
I tried twice and it didn't work. the nodes were still being created when the ansible stopped.